code/modules/mob/living/simple_animal/abnormality/waw/sphinx.dm

```diff
@@ -529,59 +452,44 @@
 	variable = TRUE
 	multiplicative_slowdown = 2
 
-// Objects - Structures
-/obj/structure/sacrifice_table
-	name = "sacrificial altar"
-	desc = "It looks impossibly ancient."
-	icon = 'ModularTegustation/Teguicons/32x32.dmi'
-	icon_state = "altar"
-	anchored = TRUE
-	density = TRUE
-	layer = TURF_LAYER
-	plane = FLOOR_PLANE
-	resistance_flags = INDESTRUCTIBLE
-	var/obj/item/showpiece = null
-
-/obj/structure/sacrifice_table/examine(mob/user)
+	//Effects
+/obj/effect/temp_visual/rockattack
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "rockattack"
+	duration = 6
+	randomdir = TRUE //random spike appearance
+	layer = ABOVE_MOB_LAYER
+
+/obj/effect/temp_visual/rockwarning
+	name = "moving rocks"
+	desc = "A target warning you of incoming pain"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "rockwarning"
+	duration = 10
+	layer = RIPPLE_LAYER	//We want this HIGH. SUPER HIGH. We want it so that you can absolutely, guaranteed, see exactly what is about to hit you.
```

Comment:
Please if you could, don't use tabs for comments. Using a singular space makes it less jumpy and upredictable.
Bonus points if you put a space after `//`, tho thats just my personal preference
```suggestion
	layer = RIPPLE_LAYER //We want this HIGH. SUPER HIGH. We want it so that you can absolutely, guaranteed, see exactly what is about to hit you.
```